triangles/triangles.py

```diff
@@ -0,0 +1,104 @@
+import sys
```

Comment:
Your Pull Request title: TRIANGLE-1021 Added triangles.py

Check result:

* Pull Request title must start with prefix in ['GENERATOR', 'LEETCODE']